src/plots/cartesian/layout_attributes.js

```diff
@@ -61,6 +61,11 @@ module.exports = {
         dflt: '-',
         role: 'info',
         editType: 'calc',
+        // we forget when an axis has been autotyped, just writing the auto
+        // value back to the input - so it doesn't make sense to template this.
+        // TODO: should we prohibit this in `coerce` as well, or honor it if
+        // someone enters it explicitly?
+        _noTemplating: true,
```

Comment:
I prevented `makeTemplate` from pulling out `axis.type` with this flag, as it seems more likely to cause problems than fix them, but I did NOT prevent `coerce` from using it if someone makes a template of manually and includes it (to say "always make this axis categorical" for example). Seem reasonable?